DuckDuckGo/MainViewController.swift

```diff
@@ -30,6 +30,10 @@ import BrowserServicesKit
 class MainViewController: UIViewController {
 // swiftlint:enable type_body_length
 
+    static var sharedInstance: MainViewController {
+        (UIApplication.shared.delegate?.window??.rootViewController as? MainViewController)!
+    }
```

Comment:
We shouldn't add helpers that promote accessing MainVC other than that it comes naturally from parent-child relationship.